stripe/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -66,7 +66,12 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
     protected val _userMessage = MutableLiveData<UserMessage?>()
     internal val userMessage: LiveData<UserMessage?> = _userMessage
 
-    abstract val newCard: PaymentSelection.New.Card?
+    /**
+     * This should be initialized from the starter args, and then from that
+     * point forward it will be the last valid card seen or entered in the add card view.
+     * In contrast to selection, this field will not be updated by the list fragment.
+     */
+    abstract var newCard: PaymentSelection.New.Card?
```

Comment:
Is this really only relevant for `PaymentOptionsActivity`? We don't need to refactor here, but if so maybe in a future PR we can pull it out of the base class.

Reply:
It is actually needed in both because the in complete if you add a new card, go back to the list, and click add again it will repopulate, and it can only do that with new card.

Reply:
Makes sense! 💪